src/styles/block/card.scss

```diff
@@ -0,0 +1,129 @@
+.card {
+  width: $card-width;
+
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 16px;
+  transition-duration: $transition-speed;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 16px 3px 0;
+  }
+
+  &__name {
+    font-family: inherit;
+    display: flex;
+
+    font-size: 12px;
+    font-weight: 500;
+
+    line-height: 18px;
+    color: $color-text;
+
+    margin: 0;
+    padding-top: 40px;
+
+    transition-duration: $transition-speed;
+  }
+
+  &:hover &__name {
+    color: $color-title-hover;
+  }
+
+  &__code {
+    font-family: inherit;
+    font-size: 10px;
+    font-weight: 400;
+
+    margin: 0;
+    padding-top: 4px;
+
+    line-height: 14px;
+    color: $color-text-small;
+  }
+
+  &__reviews {
+    font-family: inherit;
+
+    display: flex;
+    justify-content: space-between;
+
+    padding-top: 18px;
+    padding-left: 2px;
```

Comment:
The padding-left value seems arbitrary. If you are trying to align items within the container, consider using consistent padding values or flexbox alignment properties.